### ISSUE TITLE
Only one DNS name server outputted when two are supplied.

### ISSUE THREAD

**Issue by JasonSwindle** [TKCERT/pfFocus#9]
```xml
		<dnsserver>1.1.1.1</dnsserver>
		<dnsserver>1.0.0.1</dnsserver>
```

```md
| dnsserver   | 1.0.0.1                                                        |
```

Not sure if this is a know issue or not. :)

**Comment by raulcpop** [comment]
ModuleNotFoundError: No module named 'pf_focus'

I have this error after I installed it with python3

**Comment by mback2k** [comment]
@JasonSwindle your issue should be fixed with 6272213b6f698fec8dac1df5b2179c1caa290dcc. @raulcpop your issue will be tracked with #11.
